src/fides/api/ops/service/authentication/authentication_strategy_api_key.py

```diff
@@ -59,5 +61,17 @@ def add_authentication(
                     query_param.name,
                     param_val,
                 )
+        if self.body:
+            body_with_api_key: Optional[str] = assign_placeholders(
+                self.body, secrets or {}
+            )
+            api_key_request_body: Dict = json.loads(body_with_api_key or "{}")
+
+            original_request_body: Dict = json.loads(request.body or "{}")
+            original_request_body.update(api_key_request_body)
+
+            request.prepare_body(
+                data=json.dumps(original_request_body), files=None, json=True
+            )
```

Comment:
This makes sense, we originally didn't implement the body because we had concerns about how an authentication body would merge with a request body but it's good to see this working as expected here.

Reply:
agreed, i like this implementation!